neurovault/api/serializers.py

```diff
@@ -229,8 +230,7 @@ class Meta:
 class CollectionSerializer(serializers.ModelSerializer):
     url = HyperlinkedImageURL(source='get_absolute_url', read_only=True)
     owner = serializers.ReadOnlyField(source='owner.id')
-    images = ImageSerializer(many=True, source='image_set')
-    nidm_results = NIDMResultsSerializer(many=True, source='nidmresults_set')
+    images = ImageSerializer(many=True, source='basecollectionitem_set')
```

Comment:
I think I'm missing something, because I can't see `items` field neither in `CollectionViewSet`, nor in `CollectionSerializer`.